src/Components/Footer/Footer.js

```diff
@@ -1,29 +1,54 @@
 import React from "react";
+import { Link } from "react-router-dom";
 import "./Footer.css";
 
 export default function Footer() {
     return (
-        // TODO: add a tags to the links
+        // TODO: add a tag to Episodes when that is made
         <div className="footerContainer">
             <hr className="footerHR"></hr>
             <div className="footerLinksContainer">
                 <ul className="footerLinks">
                     <li className="footerLinksHeader">Quick Links</li>
```

Comment:
does it make sense to have this as a list item when it's more of a header?

Reply:
I mean more that maybe it should be structure more like this maybe:
```
<h3>Quick Links</h3>
   <ul>
      <li>...</li>
   </ul>

Reply:
because Quick Links isn't really a list item. It's more of a title for the list
But also it's such a nit pick, doesn't really matter

Reply:
It looks fine on the live site so I don't mind. But if you feel like you wanna give it a go, go for it!

Reply:
i give up